packages/router-generator/tests/generator.test.ts

```diff
@@ -40,19 +40,25 @@ async function traverseDirectory(
   }
 }
 
+function shouldUseInlineConfig(folder: string) {
+  return folder !== 'config-json';
+}
```

Comment:
Borrowing a pattern that I see in this file: [shouldThrow](https://github.com/TanStack/router/blob/656c56844983ede516b1c27fe312ad8e4261283a/packages/router-generator/tests/generator.test.ts#L202). This certainly works but I'm not confident it will scale beyond a half dozen test cases or so